.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - "3.6"
+node_js:
+  - "8"
+cache:
+  pip: true
+addons:
+  apt:
+    packages:
+    - httpie
+install:
+  - set -eo pipefail
+  - pip install flake8
+  - ./bin/build-client
+  - python setup.py install
+script:
+  - set -eo pipefail
+  - flake8 server/app/
+  - pytest -s server/test/test_filter.py server/test/test_scanpy_engine.py
+  - cellxgene &
+  - for i in {1..90}; do if http :5005/api/v0.1/initialize > /dev/null; then break; else echo "Waiting for server..."; sleep 1; fi; done
```

Comment:
You may wish to make this time-based, as that yields a bit more certainty about how long you blow through this loop. :)

Reply:
Still a loop? Or like sleep 10?

Reply:
Oops this should be fine.  I misread this.